README.md

```diff
@@ -48,6 +48,8 @@ Examples must be selected using the `--package` (or `-p`) Cargo flag.
 
 ### Winit
 
+This example has a demo [available here](https://linebender.github.io/vello) on supporting web browsers.
```

Comment:
I'd put this in the "Web" section, to leave it in context that we don't actively support the web, and that it only works on the browsers which support WebGPU